packages/vscode-extension/src/debug/commonUtils.ts

```diff
@@ -244,8 +244,8 @@ function getSettingWithUserData(jsonSelector: (jsonObject: any) => any): string
     if (isValidProject(ws)) {
       const env = getActiveEnv();
       const envJsonPath = isMultiEnvEnabled()
-        ? path.join(ws, `.${ConfigFolderName}/publishProfiles/profile.${env}.json`)
-        : path.join(ws, `.${ConfigFolderName}/env.${env}.json`);
+        ? path.join(ws, `.${ConfigFolderName}/${PublishProfilesFolderName}/profile.${env}.json`)
+        : path.join(ws, `.${ConfigFolderName}/${PublishProfilesFolderName}/env.${env}.json`);
```

Comment:
remove `${PublishProfilesFolderName}` here?

Reply:
Nice catch. Fixed and also tested local debug happy path without feature flag.